x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_logic.test.ts

```diff
@@ -0,0 +1,1006 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { resetContext } from 'kea';
+
+import { CredentialsLogic } from './credentials_logic';
+import { ADMIN, PRIVATE } from '../../constants/credentials';
+
+jest.mock('../../../shared/http', () => ({
+  HttpLogic: { values: { http: { get: jest.fn(), delete: jest.fn() } } },
+}));
+import { HttpLogic } from '../../../shared/http';
+
+function mount(defaults?) {
+  if (!defaults) {
+    resetContext({});
+  } else {
+    resetContext({
+      // the enterprise_search.app_search.credentials namespace is set in `path` in credentials_logic
```

Comment:
Since this is a newish pattern I thought I'd leave a comment here to call it out, as otherwise I think it could be confusing what this actually corresponds to.

Reply:
I think we're good to remove it now since https://github.com/elastic/kibana/pull/77698 was merged in / all front-end devs were (hopefully) on the same page!

Reply:
9111c3641bcaf8ca5e12d044a07b558c17b5e008